### PR TITLE
April blog

### PR DESCRIPTION
**Pull request by alexander-hughes**

### CODE REVIEW

**Comment by jimmytipit** [comment]
Hi Alexander - apologies for the trouble, but please note that we recently upgraded from Vuepress to Gatsby as our static site generator.  As such, there is slightly different formatting in the markup:

https://github.com/AirshipWeb/airship-website/edit/master/src/pages/blog/a-year-in-review-getting-confirmed-and-looking-ahead-to-2-0.md

If you could update your PR to match the existing blog code, we'll get it posted straight away.

**Comment by alexander-hughes** [comment]
> 
> 
> Hi Alexander - apologies for the trouble, but please note that we recently upgraded from Vuepress to Gatsby as our static site generator. As such, there is slightly different formatting in the markup:
> 
> https://github.com/AirshipWeb/airship-website/edit/master/src/pages/blog/a-year-in-review-getting-confirmed-and-looking-ahead-to-2-0.md
> 
> If you could update your PR to match the existing blog code, we'll get it posted straight away.

hi Jimmy, thanks for the feedback.  I'm trying with a rebase now, but waiting to merge until my peers have had a chance to review the rendered site.

File: src/pages/blog/airship-update-april-2020.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+- [*Airship Blog Series 2 - An Educated Evolution*](
+   https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
+- [*Airship Blog Series 1 - Evolution Towards 2.0*](
+  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+
+<br>
+
+## **SUMMARY OF MARCH VIRTUAL MEETING**
+
+As mentioned in the [*March blog*](https://www.airshipit.org/blog/airship-update-march-2020/) the Airship community came
```
Comment (drewwalters96) [inline src/pages/blog/airship-update-april-2020.md:38]: comma before "the Airship"

File: src/pages/blog/airship-update-april-2020.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+together with a virtual meeting in place of KubeCon. This was an all day event held on the 31st of March that served the
+same purpose as the originally planned face-to-face team meeting at KubeCon, the goals were:
```
Comment (drewwalters96) [inline src/pages/blog/airship-update-april-2020.md:40]: "the goals were" begins a new sentence

File: src/pages/blog/airship-update-april-2020.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  - Review of initial Airship use cases
+  - Additional Airship2 use cases
+- Session Two [*Recording*](
+https://zoom.us/rec/share/15MuEYDB53tIYJHG9WXcRpU8H735T6a80HJM-fsKzeeP9FW6Q4wvC9PyR8SpsBE?startTime=1585667419000)
+  - Metal3 discussion
+  - Explanation of CACPK and CABPK ClusterAPI providers
+  - Deep dive on CAPM3 ClusterAPI provider
+  - Kubeadm HA/Load balancing proposal
+  - Discussion on pivoting from ephemeral to target clusters
+  - Review of end to end airshipctl flow
```
Comment (drewwalters96) [inline src/pages/blog/airship-update-april-2020.md:62]: end-to-end

File: src/pages/blog/airship-update-april-2020.md (hunk 1 above)
Comment (alexander-hughes) [reply src/pages/blog/airship-update-april-2020.md:38]: done

File: src/pages/blog/airship-update-april-2020.md (hunk 2 above)
Comment (alexander-hughes) [reply src/pages/blog/airship-update-april-2020.md:40]: done

File: src/pages/blog/airship-update-april-2020.md (hunk 3 above)
Comment (alexander-hughes) [reply src/pages/blog/airship-update-april-2020.md:62]: done

**Comment by alexander-hughes** [comment]
If no negative feedback by EOD Tuesday will merge on Wednesday the 15th

**Comment by jimmytipit** [comment]
SHould I go ahead and merge this?

**Comment by mattmceuen** [comment]
LGTM!

**Pull request merged**